welcome/index.adoc

```diff
@@ -28,6 +28,7 @@ To navigate the {product-title} {product-version} documentation, you can use one
 
 * Use the navigation bar to browse the documentation.
 * Select the task that interests you from xref:../welcome/learn_more_about_openshift.adoc#learn_more_about_openshift[Learn more about {product-title}].
+* {product-title} has a variety of layered offerings to add additional functionality and extend the capabilities of a cluster. For more information, see link:https://access.redhat.com/support/policy/updates/openshift_operators[{product-title} Operator Life Cycles]
```

Comment:
Was the Operator lifecycle page intended here or should this link point to https://access.redhat.com/support/policy/updates/openshift?

Reply:
It was intentional to show the layered products, but we can mention this link as well.